renderers/src/test/java/com/pedrogomez/renderers/RVRendererAdapterTest.java

```diff
@@ -163,6 +165,28 @@ public void shouldThrowNullRendererBuiltException() {
     verify(mockedRenderer).render();
   }
 
+  @Test public void shouldSetAdapteeCollection() throws Exception {
+    RVRendererAdapter<Object> adapter = new RVRendererAdapter<Object>(mockedRendererBuilder);
+
+    adapter.setCollection(mockedCollection);
+
+    assertEquals(mockedCollection, adapter.getCollection());
+  }
+
+  @Test public void shouldBeAdapteeCollectionNotNullWhenCreateOnlyWithRendererBuilder() {
+    RVRendererAdapter<Object> adapter = new RVRendererAdapter<Object>(mockedRendererBuilder);
+
+    assertNotNull(adapter.getCollection());
```

Comment:
Instead of checking that the collection is not null, we should check that the number of items is 0. If the collection inside the ``RVRendererAdapeter`` is null is an implementation detail. We could implement the same  behaviour using a null check so my recommendation is to update this test so something like:

```java
@Test public void shouldBeEmptyWhenItsCreatedWithJustARendererBuilder() {
    RVRendererAdapter<Object> adapter = new RVRendererAdapter<Object>(mockedRendererBuilder);

    assertEquals(0, adapter.getCount());
}
```